### PR TITLE
Remove unsupported event type

### LINKED ISSUE #3: Remove event publishing
Event support will be removed from ffwd in https://github.com/spotify/ffwd/pull/153. This client should also remove events to match.

### PR DESCRIPTION
**Pull request by dmichel1**
Closes #3

### CODE REVIEW

File: src/main/proto/protocol0.proto [hunk 1]
```diff
@@ -36,33 +36,11 @@ message Metric {
...
-}
-
 message Attribute {
   required string key = 1;
   optional string value = 2;
 }
 
 message Message {
   optional Metric metric = 1;
-  optional Event event = 2;
```
Comment (dmichel1) [inline src/main/proto/protocol0.proto:67]: ah actually maybe i shouldn't remove this to remain backwards compatible? not sure how ffwd uses this.

Comment (hexedpackets) [reply src/main/proto/protocol0.proto:67]: Yeah, its pretty much never a good idea to remove a protobuf field. You can annotate it with `[deprecated=true]`.

Comment (dmichel1) [reply src/main/proto/protocol0.proto:67]: https://github.com/spotify/ffwd/blob/master/modules/protobuf/src/main/java/com/spotify/ffwd/protobuf/ProtobufDecoder.java#L111

I could do https://developers.google.com/protocol-buffers/docs/proto3#reserved

Comment (hexedpackets) [reply src/main/proto/protocol0.proto:67]: Or making it a reserved field - https://developers.google.com/protocol-buffers/docs/proto#reserved

File: src/main/proto/protocol0.proto [hunk 2]
```diff
@@ -36,33 +36,14 @@ message Metric {
...
-  repeated Attribute attributes = 9;
-}
-
 message Attribute {
   required string key = 1;
   optional string value = 2;
 }
 
 message Message {
+  reserved 2;
```
Comment (dmichel1) [inline src/main/proto/protocol0.proto:45]: @hexedpackets how does this look?

**Review by hexedpackets — APPROVED** [review]

**Pull request merged**